google/cloud/spanner/client.cc

```diff
@@ -155,6 +158,28 @@ StatusOr<CommitResult> RunTransaction(
   return client.Commit(txn, *mutations);
 }
 
+StatusOr<CommitResult> RunTransaction(
+    Client client, Transaction::ReadWriteOptions const& opts,
+    std::function<StatusOr<Mutations>(Client, Transaction)> const& f) {
+  ExponentialBackoffPolicy backoff_policy(std::chrono::milliseconds(100),
+                                          std::chrono::minutes(5), 2.0);
+  LimitedErrorCountRetryPolicy retry_policy(/*maximum_failures=*/2);
+
+  Status last_status;
```

Comment:
Perhaps this shouldn't be initialized to `OK` (maybe `IsExhausted()` is initially true).

Reply:
Fixed.